### ISSUE TITLE
The blinking page title is not accessible and not descriptive enough

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2330]
Originally reported on Google Code with ID 1578

```
The blinking title doesn't function well with screen readers and a page title of a plain
"BigBlueButton" isn't descriptive enough to tell two different tabs apart.

I will be disabling the blinking and changing the page title to the meeting title.

```

Reported by `capilkey` on 2013-07-04 22:19:13
